keras/src/backend/numpy/image.py

```diff
@@ -493,6 +493,235 @@ def affine_transform(
     return affined
 
 
+def perspective_transform(
+    images,
+    start_points,
+    end_points,
+    interpolation="bilinear",
+    fill_value=0,
+    data_format=None,
+):
```

Comment:
The signature looks good -- we can add this new function to the `keras.image` API.